index.js

```diff
@@ -68,7 +68,7 @@ module.exports = function(version, _options) {
 
             return config.join('');
         },
-        compile: function(language, step) {
+        compile: function(language, step, legIndex) {
```

Comment:
How would you indicate “final”? −1? (We would also need to add a string for “final” to the translations.)